package.json

```diff
@@ -42,7 +43,7 @@
     "tsdocs": "lerna run --scope @loopback/tsdocs build:tsdocs",
     "coverage:ci": "node packages/build/bin/run-nyc report --reporter=text-lcov | coveralls",
     "precoverage": "npm test",
-    "coverage": "open coverage/index.html",
+    "coverage": "open-cli coverage/index.html",
```

Comment:
Can we move this change to a separate PR?

Reply:
Done!